components/linear/sources/comment-created-instant/comment-created-instant.mjs

```diff
@@ -0,0 +1,15 @@
+import common from "../common/webhook-props.mjs";
+import commentCreatedInstant from "../../../linear_app/sources/comment-created-instant/comment-created-instant.mjs";
+
+/* eslint-disable pipedream/required-properties-type */
+/* eslint-disable pipedream/required-properties-name */
+
+export default {
+  ...commentCreatedInstant,
+  key: "linear-comment-created-instant",
+  description: "Emit new event when a new comment is created (OAuth). See the docs [here](https://developers.linear.app/docs/graphql/webhooks)",
+  version: "0.0.2",
```

Comment:
Can this version be synced with `commentCreatedInstant`'s version? So that both `linear` and `linear_app` are rev'ed and published together.
Will probably need this:
`// eslint-disable-next-line pipedream/required-properties-version`

**Can this change be applied to the other components as well?**

```suggestion
```